contracts/package.json

```diff
@@ -61,6 +61,5 @@
         "ts-node": "^10.0.0",
         "typechain": "^5.1.2",
         "typescript": "^4.5.5"
-    },
-    "dependencies": {}
+    }
```

Comment:
Remove this so there is no change to this file

Reply:
https://github.com/engineer-dao/platform/blob/838d776f19d287ed510fb5677f97cb37249a8fec/contracts/package.json
seems like it's notherehhttps://github.com/engineer-dao/platform/blob/838d776f19d287ed510fb5677f97cb37249a8fec/contracts/package.json
seems like it's not there anymore? I'm not sure what happened tho.